felix/bpf/ut/bpf_prog_test.go

```diff
@@ -934,6 +934,7 @@ func (pkt *Packet) handleL4() error {
 	case *layers.UDP:
 		pkt.udp = v
 		pkt.length += 8
+		pkt.udp.Length = uint16(pkt.length)
```

Comment:
@tomastigera you set it by changing `v`, I just thought to update `pkt` like the rest of fields. It should be the same. WDYT?